### PR TITLE
fixed reading PQR files from Gromacs

### LINKED ISSUE #1804: pqrparser having trouble parsing gromacs generate pqr file
### Expected behaviour
I have made some pqr file using gromacs editconf tools
`gmx editconf -f enmin.tpr -mead system.pqr`
I'm interested in using MDAnalysis to analysis it.

### Actual behaviour
``` python
import MDAnalysis as mda
from six import StringIO
pqrfile = '''REMARK    The B-factors in this file hold atomic radii
REMARK    The occupancy in this file hold atomic charges
TITLE     system
REMARK    THIS IS A SIMULATION BOX
CRYST1   35.060   34.040   38.990  90.00  90.00  90.00 P 1           1
MODEL        1
ATOM      1  O    ZR     1      15.710  17.670  23.340 -0.67  1.48           O
TER
ENDMDL'''
u = mda.Universe(StringIO(pqrfile), format='pqr')
```
Gives
Error: invalid literal for int() with base 10: '15.71'

A quick look at the code shows that the pqrparser is not expecting an atom type and that makes the pqrparser think that the segid is included in the pqr file.

### Currently version of MDAnalysis:
mda 0.17.0/py36

### PR DESCRIPTION
**Pull request by richardjgowers**
Fixes #1804

Changes made in this Pull Request:
 - can now read PQR files from gromacs

PR Checklist
------------
 - [x] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

File: package/MDAnalysis/topology/PQRParser.py [hunk 1]
```diff
@@ -94,7 +94,9 @@ class PQRParser(TopologyReaderBase):
        Now reads insertion codes and splits into new residues around these
     .. versionchanged:: 0.17.1
        Added parsing of Record types
-
+    .. versionchanged:: 0.17.1
```
Comment (orbeckst) [inline package/MDAnalysis/topology/PQRParser.py]: merge with previous version changed record

File: testsuite/MDAnalysisTests/topology/test_pqr.py [hunk 2]
```diff
@@ -67,3 +68,25 @@ def test_record_types():
...
+'''
+
+def test_gromacs_flavour():
+    u = mda.Universe(StringIO(GROMACS_PQR), format='PQR')
+
+    assert len(u.atoms) == 1
+
+    assert u.atoms[0].type == 'O'
+    assert u.atoms[0].segid == 'SYSTEM'
+    assert not u._topology.types.is_guessed
```
Comment (orbeckst) [inline testsuite/MDAnalysisTests/topology/test_pqr.py]: Add an assert for the coordinates or add a test to the coordinates tests.

**Comment by richardjgowers** [comment]
@orbeckst good catch, the different format actually shifts the coordinate positions!

**Comment by orbeckst** [comment]
@richardjgowers can you please rebase against develop – I merged PR #1858 so the tests should pass again.

**Comment by richardjgowers** [comment]
@orbeckst good to go

**Comment by orbeckst** [comment]
The Travis failure appears to be spurious.

**Pull request merged**